llvm/test/Verifier/AMDGPU/intrinsic-amdgpu-init-exec-from-input.ll

```diff
@@ -18,10 +17,3 @@ define amdgpu_ps void @init_exec_from_input_fail_not_inreg(i32 inreg %a, i32 %b)
   call void @llvm.amdgcn.init.exec.from.input(i32 %b, i32 0)
   ret void
 }
```

Comment:
Also test a case where it's not an argument (try one with a constant and one with an instruction)

Reply:
Thanks, I added another check for the case that the operand is an instruction!
Isn't the constant case covered by the first function?